src/graph/graph.people.ts

```diff
@@ -66,19 +67,21 @@ export async function getPeopleFromGroup(graph: IGraph, groupId: string): Promis
  * returns a promise that resolves after specified time
  * @param time in milliseconds
  */
-export function getEmailFromGraphEntity(entity: User | Person | Contact): string {
+export function getEmailFromGraphEntity(entity: User | Person | Contact | IDynamicPerson): string {
```

Comment:
IDynamicPerson is already `User| Person | Contact`

```suggestion
export function getEmailFromGraphEntity(entity: IDynamicPerson): string {
```

Reply:
Updated to simplify:

462e6dc